srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/ValueRetriever.java

```diff
@@ -207,6 +209,28 @@ public void execute(final Patient patient, final VariableEntry variableEntry, fi
         {
             addLabValue("PTT", patient, variableEntry, key);
         }
+    },
+    FUNCTIONAL_STATUS
```

Comment:
I think this constant should be called "ADL_NOTES", note "FUNCTIONAL_STATUS". The former is what it is actually pulling from VistA.